BUILD

```diff
@@ -193,6 +193,7 @@ genrule(
 
 pkg_npm(
     name = "npm_package",
+    package_name = "@bazel/bazelisk",
```

Comment:
is this needed?

Reply:
yes, the new rules_nodejs version complains if this isn't added.

Reply:
See https://buildkite.com/bazel/bazelisk/builds/1707